ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -747,3 +747,9 @@ def remainder(self, y, name=None):
 
     def is_floating_point(self):
         return paddle_frontend.is_floating_point(self._ivy_array)
+
+    @with_supported_dtypes(
+        {"2.5.1 and below": ("float32", "float64", "int32", "int64")}, "paddle")
+
```

Comment:
Hi @Boghdady9, please resolve the lint errors found in the CI checks at the bottom of the PR.